config.yaml

```diff
@@ -14,6 +14,8 @@ environment:
 weather:
     station: mongo
     aag_cloud:
+        product_1: 'aag-cloudsensor'
+        product_2: 'aat-metdata'
```

Comment:
I think we will want the to have lists under `weather`. One of those items ("products") is the aag and the other is the aat.

Reply:
How would I do that? Still trying to grasp how yaml works.

Reply:
It's kind of just like a python dict, but basically you want something like:

```
weather:
    web_service:
         link: 'http://site.aao.gov.au/met/metdata.dat'
         column_names:  ['Time (UTC)',        
         ...
    aag_cloud:
        serial_port: /dev/ttyUSB0
        ....
```

Reply:
Then you can get `web_data = self.cfg['weather']['web_service']`

Reply:
Yep. So I should just have `config.yaml` and put everything from `local_config.yaml` into it?

Reply:
The file should actually be `config_local.yaml` and actually shouldn't go into the repository. For now you can put the items directly into `config.yaml` but we'll actually want to think through that a bit. By putting it in the `config.yaml` (and there into the repository), you are making it so *all* people use this going forward.